community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,119 @@
+for respectful and appropriate conduct across all communication channels. For
+this purpose, we have created this code of conduct, which serves as a guide for
+interaction with the SourceCred community. It is adopted from [several well known](#license)
+Code of Conducts of open source projects.
+
+## People
+
+In the following sections:
+
+ - **we/our**: refers to the members of the SourceCred community, who exist on equal footing.
+ - **SourceCred**: refers to the general project or community.
+ - **Benevolent Dictator**: refers to @decentralion
+
+## Diversity Statement
+
+SourceCred welcomes and encourages participation in our community by people of 
+all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning, and you can read more about culture
+in the [culture](../culture) documentation pages. We strive to work together 
+as a community to realize these values.
+
+We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and more productive and motivated. A diverse community where people treat one another with respect has more potential contributors, more sources for ideas, and fewer shared assumptions that might hinder the achievement of our goals.
+
+Thus, we welcome all people to participate in SourceCred regardless of their identity
+or background.
+
+## Introduction & Scope
+
+This code of conduct should be honored by everyone who participates in the SourceCred community.
+It should be honored in any SourceCred activities including but not limited to
+online interactions (e.g. public and private mailing lists, issue trackers, discussion forums, wikis, and code repositories), meetups or conferences, and adhered to by anyone claiming affiliation 
+with the SourceCred community. This code of conduct is especially relevant for any
+group or individual that is representing the SourceCred community in an event
+(i.e., as speaker, volunteer, or other staff), and should govern standards of behaviors
+for the individual's participation in said event.
+
+This code of conduct is not exhaustive or complete, but rather serves to distill our understanding of a collaborative, inclusive community, and to provide avenues for seeking support if this
+code is breached. We should try to follow this code in spirit as much as in letter, to create a friendly and productive environment that enriches the SourceCred community.
+
+Unless explicitly specified for a particular project or initiative, this code
+of conduct serves as a global standard for all SourceCred affiliated projects, activities,
+and events. If an individual is found to violate this Code of Conduct via a [confidential report](#confidentiality), at their discretion the Benevolent Dictator may consult with other members from the community, for example through a committee, and take any action they
```

Comment:
This is exactly what I had in mind with the "cross-linking" of confidentiality.
It's subtly mentioned here right next to the actions we might take. So that will be easier to find for readers, and strengthens the reminder that reports are expected to be treated as confidential.

:+1: thanks!